packages/adblocker/src/filters/network.ts

```diff
@@ -581,14 +581,25 @@ export default class NetworkFilter implements IFilter {
             }
             break;
           case 'redirect-rule':
-          case 'redirect':
+          case 'redirect': {
             // Negation of redirection doesn't make sense
             if (negation) {
               return null;
             }
 
             // Ignore this filter if no redirection resource is specified
-            if (value.length === 0 || value.startsWith(':') || value.endsWith(':')) {
+            if (value.length === 0) {
+              return null;
+            }
+
+            // Ignore this filter if wrong priority is given
+            const priorityIndex = value.lastIndexOf(':');
```

Comment:
I'd suggest using `lastIndexOf` rather than `indexOf` for the following reasons:
- Searching from backwards can be better in most cases of redirect filter syntax
- I don't think we'll have a case but it's enables us to handle resource name with colon.